### ISSUE TITLE
pn:hasPlaceName modeled as both data and object property

### ISSUE THREAD

**Issue by dr-shorthair** [GeoscienceAustralia/Place-Names#11]
The property [pn:hasPlaceName](http://www.linked.data.gov.au/def/placenames/#hasplacename) is specified as both an `owl:ObjectProperty` (with range `pn:PlaceName`) and an `owl:DatatypeProperty` (with range `xsd:string`). This is invalid OWL as these classes are disjoint, and poor modeling practice - the literal value should have its own datatype property.

**Comment by dr-shorthair** [comment]
use `rdfs:label` for text for the time-being

**Comment by dr-shorthair** [comment]
No - `rdfs:label a owl:AnnotationProperty` so you can't apply a cardinality constraint. 
New property: 
```
:name
  rdf:type owl:DatatypeProperty ;
  rdfs:label "name" ;
  rdfs:range xsd:string ;
.
```
Exactly 1 on a `PlaceName`

**Comment by dr-shorthair** [comment]
fixed by #9
